### PR TITLE
added New Relic, setup Artillery using payloads for Mongo and PG - refactored PG data gen function to accomodate Artillery data needs

### PR DESCRIPTION
**Pull request by matgbx**
4 tickets, but they are all smaller tasks that make up this PR:

https://trello.com/c/ZTOWMbES/20-set-up-artillery-yml-file-etc-for-csv-file-payload
https://trello.com/c/rPEGAaOc/21-create-data-gen-function-for-artillery-csv-file-needs-to-work-for-both-dbs
https://trello.com/c/2G2D0NyY/18-use-artillery-to-test-primary-db-benchmark-pre-caching-postgresql
https://trello.com/c/GGRDOJSu/19-use-artillery-to-test-secondary-db-benchmark-pre-caching-mongodb

### CODE REVIEW

File: sdc_files/artillery/mongoArtilleryRun.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const gen = require('../postgres/pgDataGen');
+const { exec } = require('child_process');
+
+// generating 1 million records for artillery csv file
+gen.reservationsList(99e5, 'mongo', (data) => {
+  if (data) {
+    const path = '/Users/MatBagnall/Desktop/matHr/Immersive/senior-portion/npm-sqrd-SDC/reservations-comp/sdc_files/artillery/mongoConfig.yml';
```
Comment (mattman240) [inline sdc_files/artillery/mongoArtilleryRun.js]: even though we are running artillery locally you should make this path relative

File: sdc_files/postgres/seedPG.js [hunk 2]
```diff
@@ -17,9 +17,10 @@ const client = new Client({
 
 client.connect();
 
-fakeData.infoList(0, (done) => {
+// pass in start amount, database (e.g. 'pg'), then callback
+fakeData.infoList(0, 'pg', (done) => {
   if (done) {
-    const path1 = '/Users/MatBagnall/Desktop/matHr/Immersive/senior-portion/npm-sqrd-SDC/reservations-comp/sdc_files/postgres/seedPG.sql';
+    const path1 = '/Users/MatBagnall/Desktop/matHr/Immersive/senior-portion/npm-sqrd-SDC/reservations-comp/sdc_files/postgres/pgCommands.sql';
```
Comment (mattman240) [inline sdc_files/postgres/seedPG.js]: same thing here

**Review by mattman240 — COMMENTED** [review]
You should probably change those paths to relative before merging this

File: sdc_files/artillery/mongoArtilleryRun.js (hunk 1 above)
Comment (matgbx) [reply sdc_files/artillery/mongoArtilleryRun.js]: Thanks, will change before merging

**Pull request merged**